packages/logger/src/index.ts

```diff
@@ -0,0 +1,177 @@
+import isError = require('lodash.iserror');
+import util = require('util');
+
+type Level = 'debug' | 'error' | 'fatal' | 'info' | 'trace' | 'warn'
+
+type LoggerConstructorOptions = {
+  asyncOperationId?: string,
+  console?: Console,
+  executions?: string,
+  granules?: string,
+  parentArn?: string,
+  pretty?: boolean,
+  sender?: string,
+  stackName?: string,
+  version?: string
+}
+
+class Logger {
+  readonly #asyncOperationId: string | undefined;
```

Comment:
I'm assuming this a convention for read-only properties?

Reply:
And re: #privateVariable .... neat.

Reply:
Adding a comment as I want to re-read the weakmap private variable oddness vs update  to make sure I'm not missing something here.

Reply:
I was thinking about it last night, and I think that I'm going to go with the `private varName: string` instead of `#varName: string`. Typescript compiles them both to the same Javascript, but "private" is more likely to pass the two beers test rather than the less obvious `#`.